bin/deploy.sh

```diff
@@ -1,33 +1,34 @@
-#!/bin/bash
-
-set -e
+#!/usr/bin/env bash
 
 ENV=$1
-RELEASE_TAG=$2
+RELEASE_TAG=$(git rev-parse HEAD)
```

Comment:
Removes the need for manual setting of a version. The AWS EB version GUI has dates and what env a version is deployed to so I've found that and a SHA to identify commits is sufficient.

This is a matter of taste though!